cmd/goal/commands.go

```diff
@@ -92,8 +97,13 @@ var rootCmd = &cobra.Command{
 	Long:  `GOAL is the CLI for interacting Algorand software instance. The binary 'goal' is installed alongside the algod binary and is considered an integral part of the complete installation. The binaries should be used in tandem - you should not try to use a version of goal with a different version of algod.`,
 	Args:  validateNoPosArgsFn,
 	Run: func(cmd *cobra.Command, args []string) {
+		if versionCheck {
+			version := config.GetCurrentVersion()
+			fmt.Printf("%d\n%s.%s [%s] (commit #%s)\n%s\n", version.AsUInt64(), version.String(),
+				version.Channel, version.Branch, version.GetCommitHash(), config.GetLicenseInfo())
```

Comment:
This version printing code repeats twice in this change set plus two more times in cmd/algo[h|d]/main.go. Makes sense to implement a lib function now.